decentraland/unity-explorer
# Fix: terrain color blend rerender

## What does this PR change?
fix #748
fix #756

## How to test the changes?

1. Launch the explorer
2. Go to some world -> verify world terrain already presented after loading screen disappears
3. go back to genesis city -> verify grass in not red (see #748)
4. try this loop several times (but with jumping between stable worlds and empty genesis terrain)
5. check how it works at night
6. verify that  building at -150,-3 is not covered by bushes (it is owned parcel). Like that 
![Uploading image.png…]()


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
